Site/bc_weather_station_dashboard/website/static/map.js

```diff
@@ -106,7 +106,13 @@ function updateDataHTML(currentStationData) {
 
     // Update the HTML elements with the station's wind direction data
     if (currentStationData.HOURLY_WIND_DIRECTION) {
-        document.getElementById('wind-direction').innerHTML = currentStationData.HOURLY_WIND_DIRECTION + "&deg;";
+        // Get the wind direction in degrees
+        var windDirectionDegrees = currentStationData.HOURLY_WIND_DIRECTION;
+        // Update the text display append to the wind direction widget
+        document.getElementById('wind-direction').innerHTML = "<h5>Wind Direction " + windDirectionDegrees + "&deg;</h5>";
+        // Create a new WindArrow with the updated wind direction
+        var windArrow = new WindArrow(windDirectionDegrees);
+        windArrow.draw();
     }
```

Comment:
Do we want to move these to their own functions?

Reply:
Maybe in the future? right now map.js updates all the station elements linking them to a specific station and time such as temp, wind direction etc...